tests/unittests/Foundation/ReferenceFoundation/TestNSArray.mm

```diff
@@ -172,6 +172,12 @@ static int objectIndexInArray(NSArray* array, int value, int startingFrom, int l
         @"If NSBinarySearchingInsertionIndex is specified and no other options provided NSArray returns any equal or one larger "
         @"index than any matching object’s index.");
 
+    auto anyIndexToInsertNineteen = objectIndexInArray(array, 19, 0, array.count, NSBinarySearchingInsertionIndex);
+    ASSERT_TRUE_MSG(
+        anyIndexToInsertNineteen == array.count,
+        @"If NSBinarySearchingInsertionIndex is specified and the value is greater than any objects of the array, the returned insertion index is at the end of the array"
+        @"index than any matching object’s index.");
```

Comment:
The second @"" is a continuation of the sentence and, to me, does not lexically parse well. It at least needs correct spacing... Was this just copied from above?